SKClient/Sources/Client.swift

```diff
@@ -397,76 +397,82 @@ extension Client {
 // MARK: - Files
 extension Client {
     func processFile(_ event: Event) {
-        guard
-            let file = event.file,
-            let id = file.id
-        else {
-            return
-        }
-        if let comment = file.initialComment, let commentID = comment.id {
-            if files[id]?.comments[commentID] == nil {
-                files[id]?.comments[commentID] = comment
+        for file in event.files {
+            guard
+                let id = file.id
+            else {
+                return
```

Comment:
I think this should be `continue` and not return? Otherwise you'll miss subsequent files if you hit one that is corrupted.

Reply:
Doh, yep, thank you for catching that. 😄 I've fixed all of these.